frontend/src/routes/(authenticated)/(dashboards)/admin/+page.svelte

```diff
@@ -50,44 +62,93 @@
     )
-    .slice(0, projectSearch ? undefined : 10);
+    .slice(0, projectLimit);
   $: userSearchLower = userSearch.toLocaleLowerCase();
+  $: userLimit = userSearch ? Infinity : 10;
   $: users = $allUsers
     .filter(
       (u) =>
         !userSearch ||
         u.name.toLocaleLowerCase().includes(userSearchLower) ||
         u.email.toLocaleLowerCase().includes(userSearchLower)
     )
-    .slice(0, userSearch ? undefined : 10);
+    .slice(0, userLimit);
+
+  async function softDeleteProject(project: (typeof projects)[0]): Promise<void> {
+    const result = await deleteProjectModal.open(project.name, async () => {
+      const { error } = await _deleteProject(project.id);
+      return error?.message;
+    });
+    if (result.response === DialogResponse.Submit) {
+      notifyWarning($t('delete_project_modal.success', { name: project.name, code: project.code }));
+    }
+  }
+
+  async function refreshPage(): Promise<void> {
+    await goto(`?${toSearchParams(options)}`, {
+      replaceState: true,
+      noScroll: true,
+      keepFocus: true,
+    });
+  }
 </script>
 
 <svelte:head>
   <title>{$t('admin_dashboard.title')}</title>
 </svelte:head>
-
 <main class="flex justify-center">
   <div class="grid lg:grid-cols-2 grid-cols-1 gap-10">
     <div>
       <span class="text-xl flex gap-4">
         {$t('admin_dashboard.project_table_title')}
-        <Badge>{projectSearch ? projects.length : $allProjects.length}</Badge>
+        <Badge>
+          <span class="inline-flex gap-2">
+            {projects.length}
+            <span>/</span>
+            {$allProjects.length}
+          </span>
+        </Badge>
       </span>
 
-      <Input
-        type="text"
-        label=""
-        placeholder={$t('admin_dashboard.filter_placeholder')}
-        autofocus
-        bind:value={projectSearch}
-      />
+      <div class="flex items-end gap-4">
+        <Input
+          type="text"
+          label=""
+          placeholder={$t('admin_dashboard.filter_placeholder')}
+          autofocus
+          bind:value={projectSearch}
+        />
+        <div class="dropdown dropdown-end">
+          <!-- svelte-ignore a11y-label-has-associated-control -->
+          <label tabindex="-1" class="btn flex flex-nowrap gap-2" class:text-success={options.showDeletedProjects}>
+            {$t('admin_dashboard.options')}
+            <span class="i-mdi-dots-vertical text-xl" />
+          </label>
+          <ul tabindex="-1" class="dropdown-content menu bg-base-200 shadow rounded-box z-10">
+            <li>
+              <div class="whitespace-nowrap">
+                <label class="cursor-pointer label gap-4">
+                  <span class="label-text">{$t('admin_dashboard.show_delete_projects')}</span>
+                  <input
+                    bind:checked={options.showDeletedProjects}
+                    type="checkbox"
+                    class="toggle toggle-error"
+                    on:change={refreshPage}
```

Comment:
I'm a little worried that this code requires the bind to set the value before refresh is called. That feels very fragile. Is there another way to ensure the order doesn't change (due to a svelte change which switches the order these events fire).

Reply:
I looked into this, turns out it's actually documented: the order of the attributes determines the order of events. `bind:checked` is declared first, so the value will be ready when `on:change` fires. So, I'd like to leave this the way it is.

Docs: https://svelte.dev/docs/element-directives#bind-property
Discussion: https://github.com/sveltejs/svelte/issues/4616
![image](https://github.com/sillsdev/languageforge-lexbox/assets/12587509/44ec2b3e-92b9-4cfe-ba4a-018cc2b34ab9)